build/test-conda.ps1

```diff
@@ -37,7 +37,7 @@ function Test-CondaPackage {
 
 }
 
-Get-ChildItem $Env:CONDA_OUTDIR -Filter "*.tar.bz2" -Recurse | Test-CondaPackage;
+Get-ChildItem $Env:CONDA_OUTDIR -Filter "*.tar.bz2" -Recurse | %{ Test-CondaPackage $_.FullName }
```

Comment:
The `process` block in `Test-CondaPackage` should enable it to pull packages from the PowerShell pipeline, while sharing one call to the `begin` block. I'd suggest changing back to using the pipeline but using `Select-Object` to pick out `FullName`.

```suggestion
Get-ChildItem $Env:CONDA_OUTDIR -Filter "*.tar.bz2" -Recurse | Select-Object -ExpandProperty Full Name | Test-CondaPackage
```